arch/arc/core/fault_s.S

```diff
@@ -302,9 +302,10 @@ exc_nest_handle:
 	st _CAUSE_RIRQ, [r2, _thread_offset_to_relinquish_cause]
 	/* note: Ok to use _CAUSE_RIRQ since everything is saved */
 
-	ld_s r2, [r1, _kernel_offset_to_ready_q_cache]
+	mov r2, r0
```

Comment:
We merge this one, but this patch, here, needs to be reviewed, @abrodkin @ruuddw @vonhust. Please, submit a fix if required.

Reply:
Thanks @Ioannisg for merging this one with the deadline of 2.0 merge window closing!
Btw, I already reviewed and approved this PR (including analysis and a remark on the unrelated CI failure).
What's your concern about this patch?

Reply:
I see this is an update Wayne pushed as part of this PR. Looks ok to me. (I think the issue wasn't in the original PR but due to some other PRs/rebasing).

Reply:
@ruuddw no, this particular change was in a commit that was pushed after you approved, as far as I could see.